src/components/report-technical-issue/template.njk

```diff
@@ -1,7 +1,7 @@
 {% set language %}{% if params.language == 'cy' %}cy{% else %}en{% endif %}{% endset %}
-
+{% set serviceId = params.serviceId or params.serviceCode %}
```

Comment:
Just wondering does Nunjucks evaluate this left to right? If servceId and serviceCode is passed then does it just pick up serviceId?

Reply:
Good question. I've added a test for this edge case. serviceId takes precedence.